sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 5.0.0 (2020-07-27)
+
+- This release is to correct the package version to be 5.0.0 instead of 1.0.0.
```

Comment:
I would add some context here about why we are making this change. I would say something like "Updated the package version to 5.0.0 in order to reduce confusion with older versions of the Azure Text Analytics SDKs."

Reply:
Ha! Beat ya by 60 seconds! https://github.com/Azure/azure-sdk-for-js/pull/10297#discussion_r461115634

Reply:
done.